lib/api/v1/forms/authentication.go

```diff
@@ -21,7 +21,7 @@ func (f AuthenticationForm) Validate() (valid bool, err error) {
 	err = validation.ValidateStruct(&f,
 		validation.Field(&f.ClientID, validation.Required),
 		validation.Field(&f.ClientSecret, validation.Required),
-		validation.Field(&f.Email, validation.Required, is.Email),
+		validation.Field(&f.Email, validation.Required, is.EmailFormat),
```

Comment:
From [this reference](https://github.com/go-ozzo/ozzo-validation/blob/master/is/rules.go#L128-L131), the difference between `is.Email` and `is.EmailFormat` is the `is.Email` check the existence of the MX record for the email domain while the `is.EmailFormat` only check the formatting. In the real situation, I would prefer to use `is.Email` but currently the faker library we use on testing doesn't support faking email with real domain otherwise we'll need to add a helper function to populate the email by ourself